modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -40,7 +40,7 @@
             v-if="!!items.length"
             :selected-state="selectAllState"
             :total-pages="totalPages" :current-page="currentPage"
-            :filters="options.filters" :labels="moduleLabels"
+            :filters="toolbarFilters" :labels="moduleLabels"
```

Comment:
I don't thiunk you're passing the actual choices for the filters through yet, but I see some support for getting those choices, so I'm not sure whether to ding you here or not?

Reply:
I'm not. The ticket I linked above is solely about displaying the tags.